lua/packet.lua

```diff
@@ -708,4 +817,89 @@ end
 
 ffi.metatype("struct rte_mbuf", pkt)
 
+
+---------------------------------------------------------------------------
+---- Protocol Stacks
+---------------------------------------------------------------------------
+
+pkt.getRawPacket = packetCreate()
+
+pkt.getEthernetPacket = packetCreate("eth")
+pkt.getEthPacket = pkt.getEthernetPacket
+pkt.getEthernetVlanPacket = packetCreate({"eth", "eth", subType = "vlan"})
```

Comment:
would it be possible to allow {"eth", subType = "vlan"} if using the default name "eth"?